report/builder.go

```diff
@@ -0,0 +1,12 @@
+package report
+
+// Builder is a utility for constructing test reprots.
```

Comment:
```suggestion
// Builder is a utility for constructing test reports.
```

Reply:
This type has been removed, at least for now.